recipe/patch_yaml_utils.py

```diff
@@ -260,7 +265,7 @@ def _apply_patch_yaml(patch_yaml, record, subdir, fn):
                 elif not depends and subk in record:
                     del record[subk]
 
-            elif k == "remove_track_feature":
+            elif k == "remove_track_features":
```

Comment:
I renamed this field for consistency with the other instructions, where the convention seemed to be `{action}_{repodata-field}`.

Reply:
Yes thank you! This was bothering me yesterday!